src/parcsr_ls/par_mgr_device.c

```diff
@@ -37,7 +37,7 @@ void hypreDevice_extendWtoP( HYPRE_Int P_nr_of_rows, HYPRE_Int W_nr_of_rows, HYP
 
 HYPRE_Int
 hypre_MGRBuildPDevice(hypre_ParCSRMatrix  *A,
-                      HYPRE_Int           *CF_marker_host,
+                      HYPRE_Int           *CF_marker,
```

Comment:
@oseikuffuor1 Please double check if this is OK.

Reply:
Looks good!